sw/device/lib/arch/device_fpga_nexysvideo.c

```diff
@@ -11,7 +11,11 @@
 
 const device_type_t kDeviceType = kDeviceFpgaNexysVideo;
 
-const uint64_t kClockFreqHz = 50 * 1000 * 1000;  // 50MHz
+const uint64_t kClockFreqCpuHz = 50 * 1000 * 1000;  // 50MHz
+
+const uint64_t kClockFreqPeripheralHz = 50 * 1000 * 1000;  // 50MHz
+
+const uint64_t kClockFreqUsbHz = 48 * 1000 * 1000;  // 50MHz
```

Comment:
nit: comment reads 50MHz (here and below)

Reply:
ah thanks for catching that